metaflow/runner/utils.py

```diff
@@ -40,14 +41,21 @@ def clear_and_set_os_environ(env: Dict):
     os.environ.update(env)
 
 
+def check_process_status(command_obj: "CommandManager"):
```

Comment:
are the quotes required on the type?

Reply:
hmm, it is only imported when we do:

```
if TYPE_CHECKING:
    from .subprocess_manager import CommandManager
```

I usually put quotes on custom types / class names etc.